sftpgo/helm/sftpgo/deps.yaml

```diff
@@ -0,0 +1,30 @@
+apiVersion: plural.sh/v1alpha1
+kind: Dependencies
+metadata:
+  application: true
+  description: Deploys sftpgo crafted for the target cloud
+spec:
+  dependencies:
+  - type: helm
+    name: bootstrap
+    repo: bootstrap
+    version: '>= 0.5.1'
+  - type: helm
+    name: ingress-nginx
+    repo: ingress-nginx
+    version: ">= 0.1.2"
+  - type: terraform
+    name: aws
+    repo: sftpgo
+    version: '>= 0.1.0'
+    optional: true
+  - type: terraform
+    name: azure
```

Comment:
Remove azure and gcp terraform since we don’t need any cloud specific terraform.